src/components/widgets/hr-salaries-summary/hr-salaries-summary.less

```diff
@@ -1,3 +1,8 @@
-.analytics .widget-item .content.hr-salaries-summary {
-  .expandable-filterable-widget();
-}
\ No newline at end of file
+.analytics .widget-item {
+  .content {
+    height: auto !important;
```

Comment:
No, this should be fixed another way (without using !important).
Plus, be careful there: you are applying `height: auto !important;` to `.analytics .widget-item .content` => it will be applied to all the widgets, not only salaries summary...